src/devices/Ads1115/Ads1115.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    /// Analog-to-Digital Converter ADS1115
+    /// </summary>
+    public class Ads1115 : IDisposable
+    {
+        private I2cDevice _sensor = null;
+
+        private readonly byte _adcMux;
+        private readonly byte _adcPga;
+        private readonly byte _adcRate;
+
+        /// <summary>
+        /// Constructor
```

Comment:
NIT: Can we provide a more descriptive comment here? Something like: `Initializes a new Ads1115 device connected through I2c with specified Multiplexer configuration, Programmable Gain Amplifier and data rate` or something like that.